build/templates/tox-driver.ini.mako

```diff
@@ -0,0 +1,71 @@
+<%
+    import build.helper as helper
+
+    config = template_parameters['metadata'].config
+    module_name = config['module_name']
+    if config['supports_nitclk']:
+        nitclk_env = 'py38-{}-nitclk_wheel,'.format(module_name)
+    else:
+        nitclk_env = ''
+%>\
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox -c tox-driver.ini" from the driver directory.
+[tox]
+envlist = ${nitclk_env}py{35,36,37,38,py3}-${module_name}-system_tests
+skip_missing_interpreters=True
+ignore_basepython_conflict=True
+# We put the .tox directory outside of the workspace so that it isn't wiped with the rest of the repo
+toxworkdir = ../../../.tox
+
+[testenv]
+description =
+% if config['supports_nitclk']:
+    ${module_name}-nitclk_wheel: Build the nitclk wheel
+% endif
+    ${module_name}-system_tests: Run ${module_name} system tests (requires driver runtime to be installed)
```

Comment:
We have the name of the driver runtime in metadata, can you make it so it says:

>(requires the NI-FGEN driver runtime to be installed)

?

Reply:
Added driver name